### PR TITLE
Prantik

### PR DESCRIPTION
**Pull request by prantikseal**
Please Give me a feedback of the Html so that I can push it to main

### CODE REVIEW

**Comment by prantikseal** [comment]
screencapture-127-0-0-1-5500-Projects-team-2-first-frontend-project-starter-files-folder-starter-files-app-design-html-2022-09-17-01_51_50

**Comment by aakashjangra** [comment]
Lgtm. What's wrong with the logo img?

File: team-2-first-frontend-project/starter-files-folder/starter-files/app-design.html [hunk 1]
```diff
@@ -1,58 +1,104 @@
...
-  <ul>
-    <li>Our Company</li>
-    <li>Locations</li>
-    <li>Contact</li>
-  </ul>
-  App design
-  Our mobile designs bring intuitive digital solutions to your customers right at their fingertips.
+  <header>
+    <ul>
+      <li><img src="./assests/" alt="logo" class="logo"></li>
```
Comment (ranitasahaa1230) [inline team-2-first-frontend-project/starter-files-folder/starter-files/app-design.html:17]: -  Make another div tag for the logo and keep the logo outside of the  ul tag
- You can also use a nav tag  and inside the nav tag put the ul tag
- The logo image is not displayed in the header as well in the footer. Do check it once.

**Review by ranitasahaa1230 — COMMENTED** [review]
Suggested some changes. Rest, looks good to me

**Comment by prantikseal** [comment]
> Lgtm. What's wrong with the logo img?

the logo needs to be updated

File: team-2-first-frontend-project/starter-files-folder/starter-files/app-design.html (hunk 1 above)
Comment (prantikseal) [reply team-2-first-frontend-project/starter-files-folder/starter-files/app-design.html:17]: Thanks for the detailed feedback It has been updated and been merged

**Pull request merged**